src/modules/commander/Commander.hpp

```diff
@@ -204,6 +203,8 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 	systemlib::Hysteresis	_auto_disarm_landed{false};
 	systemlib::Hysteresis	_auto_disarm_killed{false};
 
+	bool _print_msg_once{false};
```

Comment:
I'd rename it to `_print_avoidance_msg_once` otherwise it's not clear what this is referring to.

Reply:
Done in 384cfc3